src/modules/ens/selectors.ts

```diff
@@ -20,6 +20,12 @@ export const getENSByWallet = createSelector<RootState, ENS[], string | undefine
   ensList.filter(ens => isEqual(ens.address, address))
 )
 
+export const getENSUsedAsAlias = createSelector<RootState, ENS[], string | undefined, ENS[]>(
```

Comment:
This naming is not necessarily bad but what do you think about being a bit more specific and calling it `getAlias(es)` ? Also echoing the comment above, is it posible for a user to have more than one alias? if not this could return a single `ENS`

By the way, I know the `isAlias` we've added on a previous PR lives on `ENSListPage`, but we can always move it to `ens/utils`

Reply:
Renamed to getAliases, based in my comment to the comment above! thanks!